cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -220,14 +223,16 @@ impl LockedPackageCatalog {
             version,
         } = package;
 
-        let outputs = outputs
-            .into_iter()
-            .map(|output| (output.name, output.store_path))
-            .collect();
+        let outputs = outputs.map(|outputs| {
+            outputs
+                .into_iter()
+                .map(|output| (output.name, output.store_path))
+                .collect()
+        });
 
         let priority = descriptor.priority.unwrap_or(DEFAULT_PRIORITY);
```

Comment:
Should this stay an option even in the lockfile?
seems we keep most other options now, (which I'm already curious about, see my comment on #1392)